embassy-stm32/src/usb/mod.rs

```diff
@@ -16,6 +16,7 @@ fn common_init<T: Instance>() {
 
     // On the H7RS, the USBPHYC embeds a PLL accepting one of the input frequencies listed below and providing 48MHz to OTG_FS and 60MHz to OTG_HS internally
     #[cfg(stm32h7rs)]
+    #[cfg(all(stm32u5, peri_usb_otg_hs))]
```

Comment:
ah it's failing because multiple `cfg`s are ANDed so this code is actually never compiled (no chip is H7RS and U5 at the same time!).

this should work: `#[cfg(any(stm32h7rs, all(stm32u5, peri_usb_otg_hs)))]`

Reply:
Oh I always thought multiple `cfg` statements are ORed. Let me quickly fix the `cfg` statement in this PR and then it should be good to go. @mubes please also test on your hardware once to see if everything still works to your liking as well as it's your PR after all.